core/src/com/unciv/ui/victoryscreen/VictoryScreen.kt

```diff
@@ -175,22 +178,84 @@ class VictoryScreen(val worldScreen: WorldScreen) : PickerScreen() {
 
     private fun setCivRankingsTable() {
         val majorCivs = gameInfo.civilizations.filter { it.isMajorCiv() }
-        val civRankingsTable = Table().apply { defaults().pad(5f) }
+
+        contentsTable.clear()
+        
+        if (UncivGame.Current.settings.useDemographics) contentsTable.add(buildDemographicsTable(majorCivs))
+        else contentsTable.add(buildRankingsTable(majorCivs))
+    }
+
+    enum class RankLabels { Demographic, Rank, Value, Best, Average, Worst}
+    private fun buildDemographicsTable(majorCivs: List<CivilizationInfo>): Table {
+        val demographicsTable = Table().apply { defaults().pad(5f) }
+        for (rankLabel in RankLabels.values())   {
+            if (rankLabel == RankLabels.Demographic) {
+                val demoLabel = Table().apply { defaults().pad(5f) }
+                demoLabel.add(rankLabel.name.toLabel()).row()
+                demoLabel.addSeparator().fillX()
+                demographicsTable.add(demoLabel)
+                for (category in RankingType.values()) {
+                    val headers = Table().apply { defaults().pad(5f) }
+                    headers.add(category.name.replace('_', ' ').toLabel()).row()
+                    headers.addSeparator().fillX()
+                    demographicsTable.add(headers)
+                    }
+            } else {
+                demographicsTable.row()
+                demographicsTable.add(rankLabel.name.toLabel())
+
+                for (category in RankingType.values()) {
+                    @Suppress("NON_EXHAUSTIVE_WHEN") // RankLabels.Demographic treated above
+                    when (rankLabel) {
+                        RankLabels.Rank -> { // finds the current player's rank in each category
+                            val rank = majorCivs.filter{ it.isAlive() }.sortedByDescending { it.getStatForRanking(category) }.indexOfFirst { it.civName == gameInfo.currentPlayerCiv.civName } + 1
```

Comment:
`it.civName == gameInfo.currentPlayerCiv.civName` == `it == gameInfo.currentPlayerCiv` ? the civ instances are not cloned while the player can interact, so even a `===` should see the identity?? Dunno. I'd just check in the debugger.

Reply:
And caching `majorCivs.filter{ it.isAlive() }` outside the loops could also shorten the code here. Plus - reuses.

Reply:
Cached filtered majorCivs outside the loop for reuse and simplified player equality check.